### ISSUE TITLE
Add support for video files in response

### ISSUE THREAD

**Issue by ericpullen** [aws-solutions/qnabot-on-aws#24]
Would love to have video file support for Show devices as part of the response (just like we do images today).  I'm thinking we could continue to use the image link as-is, but augment it so that if it sees a .mp4 extension it will render as video instead of picture on the Echo Show.

**Comment by JohnCalhoun** [comment]
It is hard to parse urls in answers generally, you cant tell the intent of the url: should it be a click link? an attachment? show as an image? 

what if we provided a video field in the QA document similar to the response card?  So the video is an alternative representation and not derived from the text answer and would have a separate field in the Designer UI.

**Comment by ericpullen** [comment]
I would be fine with just using a different video field. The only question I would have is how we determine if both video and image are present, which is shown (or do we do a sequential for image first then video?)

**Comment by JohnCalhoun** [comment]
#38

**Comment by JohnCalhoun** [comment]
this will be fixed when markdown support comes
